src/components/Slider/types/SliderTypes.ts

```diff
@@ -1,5 +1,59 @@
+import { AnyAction } from 'redux';
+import { SliderActions } from '../redux/sliderActions';
+
+interface LaunchMission {
+  description: string;
+}
+
 export interface Slides {
+  image: string;
+  net: string;
+  id: string;
   name: string;
-  date: string;
-  imageURL: string;
+  mission: LaunchMission;
+}
+
+export interface SliderLoadData extends AnyAction {
+  type: SliderActions.SLIDER_LOAD_DATA;
+}
```

Comment:
remove AnyAction and don't use destructuring in the reducer. Error with types will be fixed

Reply:
fixed